ios/AppLovinMAXNativeAdView.m

```diff
@@ -263,6 +261,8 @@ - (void)setMediaView:(NSNumber *)tag
     
     [self.clickableViews addObject: view];
```

Comment:
This just makes sure that the media view container stays clickable right? Or am I reading wrong?

Reply:
We should match Java as well and have both platforms be consistent.